_posts/2016-06-02-tour.md

```diff
@@ -3,19 +3,58 @@ layout: default
 title: Tour - SOCVR
 ---
 
-# TOUR
+# What this Tour is
 
-The Stack Overflow Close Vote Reviewers is just a chat room on Stak Overflow. So [**be nice**](http://stackoverflow.com/help/be-nice) also applies here!
+If you're from the [Home]({% post_url index.md %}) tab and you'd like to join SOCVR, read this tour and it will guides you what you have to know. Now, read below!
 
-## ALLOWED
 
- - You can post a [\*-pls] request, *but* please use the following format: "[\*-pls] **REASON** post link" (\*close/reopen/delete/undelete)
-   - These requests should NOT be a habit and only be used when needed
-   - Please don't spam the room with requests
+# What SOCVR is
 
-## NOT ALLOWED
+The [SO Close Vote Reviewers](http://chat.stackoverflow.com/rooms/info/41570/so-close-vote-reviewers) is a team, the main goal of ours is that **close the bad questions as quickly as possible**.
```

Comment:
Technically, the primary goal is not to close things _quickly_, just to close stuff that needs to be closed. It used to be that we focused just on the CV queue, but lately we've expanded on CV requests and bot-assisted reviews.